modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSActionDropdownItemFilter.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.model;
+
+import java.util.HashMap;
+
+/**
+ * @author Marco Leo
+ */
+public class FDSActionDropdownItemFilter extends HashMap<String, Object> {
```

Comment:
`FDSActionDropdownItemVisibilityFilter`?

Reply:
I'd like to keep it simple in case we want to reuse this class for other types of filters in the future, not related to visibility